src/main.rs

```diff
@@ -39,23 +41,49 @@ struct Args {
     #[arg(long)]
     timeout_sec: u64,
 
+    // How long to cool off before runs
+    #[arg(long, default_value_t = 0)]
+    cooloff_sec: u64,
+
     #[arg(long, default_value_t = 5)]
     times_to_run: u32,
 
-    #[arg(long)]
+    #[arg(long, default_value_t = String::new())]
     cgroup_path: String,
```

Comment:
nit: if you want to make an argument optional you can have it at the type level
```suggestion
    cgroup_path: Option<String>,
```

Reply:
Great! I understand but I was focusing on getting things to work regardless. In fact my serious tests will be *with* cgroup isolation - that is, if the java start up race doesn't prevent it.